### DISCUSSION TITLE
added some important functions

### DISCUSSION THREAD

**Discussion by cbikash** [cbikash/laravel-repository#3] [Announcements]
## What's Changed
* added functions which helps to return a query builder with associated with table by @cbikash in https://github.com/cbikash/laravel-repository/pull/2

## New Contributors
* @cbikash made their first contribution in https://github.com/cbikash/laravel-repository/pull/2

**Full Changelog**: https://github.com/cbikash/laravel-repository/compare/v1.0.0...v1.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/cbikash/laravel-repository/releases/tag/v1.0.2'>added some important functions</a>.</em>

**Comment by acharyakeshab7871** [comment]
Good job bro
